### PR TITLE
Create initial routes, render reminders

### LINKED ISSUE #1: See All Reminders
When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

* Each reminder should have a class of .spec-reminder-item.

* There is a failing test set up in tests/acceptance/reminder-list-test.js.

**Hot Reminder**: If you put “Closes #1” (assuming the issue is marked #1) in the pull request, your commit, or anywhere in the body of your PR, it will automatically close this issue when it successfully merged into master.

### PR DESCRIPTION
**Pull request by bekahlundy**
## Purpose

- When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

- Each reminder should have a class of .spec-reminder-item.

- There is a failing test set up in tests/acceptance/reminder-list-test.js.

[Issue #1](https://github.com/turingschool-projects/1610-remember-1/issues/1)

## Approach

We had to build an app with our boilerplate. We created appropriate routes, templates, and model and connected them. 

### Learning

We primarily referenced the ember docs. 

[Getting started with Ember](https://guides.emberjs.com/v2.11.0/getting-started/quick-start/)
[Ember tuturial](https://guides.emberjs.com/v2.11.0/tutorial/ember-cli/)

### Open Questions and Pre-Merge TODOs

- [x] Set up routes and model for our application
- [x] Change Ember version to 2.11.0
- [x] Reminders render on page
- [x] Add class .spec-reminder-item to li tags
- [x] Route on page load is /
- [x] Tests run and fail

### Test coverage 

There were two tests written for us. There is a test to check if there are 5 reminders rendering on the homepage, and a test to check when something is clicked.

### Merge Dependencies and Related Work

N/A ???

### Follow-up tasks

- [Issue #2](https://github.com/turingschool-projects/1610-remember-1/issues/2)

#### Before
screen shot 2017-02-08 at 3 13 14 pm
#### After
screen shot 2017-02-08 at 3 14 46 pm
screen shot 2017-02-08 at 3 17 07 pm

close #1

### CODE REVIEW

File: app/routes/reminders.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return ['bekah needs to get more banana chips', 'josh needs to do a cartwheel', 'Bekah needs to get out of her funk', 'We both need to listen to some funky music', 'make money']
```
Comment (martensonbj) [inline app/routes/reminders.js]: 1) Let's get bekah out of her funk AND more cartwheels would be good. 2. Let's use Mirage to grab these models, we don't need to hardcode them. Check out [docs for how to findAll records from a store](https://guides.emberjs.com/v2.11.0/models/finding-records/#toc_retrieving-multiple-records) -- this call will trigger Mirage to hand you whatever it's created instead.

File: app/templates/application.hbs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1>hello</h1>
```
Comment (martensonbj) [inline app/templates/application.hbs]: I'm assuming you meant to delete this h1 - unless that's an intentional title in which case let's capitalize it.

File: app/templates/components/reminders-list.hbs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1>{{title}}</h1>
+
+<ul>
+  {{#each model as |reminder|}}
+  <li class='spec-reminder-item'>{{reminder}}</li>
```
Comment (martensonbj) [inline app/templates/components/reminders-list.hbs]: indent please :)

File: app/templates/index.hbs [hunk 4]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (martensonbj) [inline app/templates/index.hbs]: We don't need this file at all - can be deleted

File: tests/integration/components/reminders-list-test.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/integration/components/reminders-list-test.js:1]: This is all boilerplate code - lets add some sort of assertion that indicates you know what this file does. In this case I would at least test that there is an h1, and 5 elements with the class of '.spec-reminder-item'.

File: tests/unit/routes/index-test.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: We won't need any route tests. Routes are generally tested in acceptance tests when you run the line `visit('/')` and then assert that the current URL is what you expect. We can delete this file.

File: tests/unit/routes/reminders-test.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/reminders-test.js]: See above comment on route tests.

File: app/templates/application.hbs (hunk 2 above)
Comment (martensonbj) [reply app/templates/application.hbs]: Also we can remove comments.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Few things! Good work 👍

**Comment by martensonbj** [comment]
Status on these changes? Checking in to make sure youv'e seen them. Make sure any changes related to this PR are pushed up to the same branch this PR lives on.

**Comment by bekahlundy** [comment]
@martensonbj Here are our changes! ✨

**Comment by bekahlundy** [comment]
Hey @Tman22 @martensonbj , we made the changes that Taylor had talked to us about. Deleted some files and extra code and only included issue 1 in this pr.

File: tests/acceptance/reminder-list-test.js [hunk 8]
```diff
@@ -13,12 +13,12 @@ test('viewing the homepage', function(assert) {
   visit('/');
 
   andThen(function() {
-    assert.equal(currentURL(), '/');
-    assert.equal(Ember.$('.spec-reminder-item').length, 5);
+    assert.equal(currentURL(), '/', 'curren turl is root');
```
Comment (rrgayhart) [inline tests/acceptance/reminder-list-test.js:16]: @bekahlundy - This is misspelled

**Review by martensonbj — APPROVED** [review]

File: tests/acceptance/reminder-list-test.js (hunk 8 above)
Comment (martensonbj) [reply tests/acceptance/reminder-list-test.js:16]: @rrgayhart that was actually my fault - they're fixing it in the next PR

**Pull request merged**